CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ Datadog Agent 6 has been officially released and the release notes can be found
 Changes
 =======
 
+# 5.32.5 / 2019-10-14
+
+**Windows and Source Install**
```

Comment:
I don't think it makes sense to release for `Source Install` since the change only affects Windows, and the `Source Install` is only available on Unix platforms on Agent v5.

Let's mention Windows only instead?

```suggestion
**Windows only**
```